server/src/test/java/io/crate/integrationtests/ColumnPositionsIntegrationTest.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.integrationtests;
+
+import org.junit.Test;
+
+import static io.crate.testing.TestingHelpers.printedTable;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ColumnPositionsIntegrationTest extends SQLIntegrationTestCase {
```

Comment:
Could we maybe integrate position number assertions into existing tests which add dynamic columns and alter table add column tests? I don't think it's warranted to test this in isolation. Asserting that a column was added corretly makes more sense than asserting that one particular property of an added column was set correctly.